### PR TITLE
Upgrade transaction structure: add Timestamp field

### PR DESCRIPTION
**Pull request by ggicci**

### CODE REVIEW

**Comment by auxten** [comment]
Fix PR name according to our coding guide

File: blockproducer/interfaces/mixins.go [hunk 1]
```diff
@@ -20,7 +20,16 @@ package interfaces
...
-	TxType TransactionType
+	TxType    TransactionType
+	Timestamp int64
+}
+
+// NewTransactionTypeMixin returns new instance.
+func NewTransactionTypeMixin(txType TransactionType) *TransactionTypeMixin {
+	return &TransactionTypeMixin{
+		TxType:    txType,
+		Timestamp: 0,
```
Comment (leventeliu) [inline blockproducer/interfaces/mixins.go]: Use `time.Now`.

**Review by leventeliu — APPROVED** [review]
LGTM.

**Review by xq262144 — APPROVED** [review]
Approve

**Pull request merged**